<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Site</class>
 <widget class="QWidget" name="Site">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_datas">
     <property name="title">
      <string>Datas</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkBox_6">
        <property name="text">
         <string>TM phase</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_11">
        <property name="text">
         <string>real(Zxx)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_1">
        <property name="text">
         <string>TE apparent resistivity (log10)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox_12">
        <property name="text">
         <string>imag(Zxx)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>TE phase</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="checkBox_7">
        <property name="text">
         <string>reserved for TM Ez/Ey tipper</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>imag(tipper) Hz/Hy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_10">
        <property name="text">
         <string>TM apparent resistivity (linear)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>real(tipper) Hz/Hy</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="checkBox_15">
        <property name="text">
         <string>real(Zyx)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="checkBox_18">
        <property name="text">
         <string>imag(Zyy)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox_13">
        <property name="text">
         <string>real(Zxy)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkBox_16">
        <property name="text">
         <string>imag(Zyx)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBox_14">
        <property name="text">
         <string>imag(Zxy)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string>TM apparent resistivity (log10)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_Reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="checkBox_8">
        <property name="text">
         <string>reserved for TM Ez/Ey tipper</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="checkBox_17">
        <property name="text">
         <string>imag(Zyy)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="checkBox_9">
        <property name="text">
         <string>TE apparent resistivity (linear)</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_import_Data">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_sites">
     <property name="title">
      <string>Sites</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1" colspan="3">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_del_site">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_export_site">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_edit_site">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_add_site">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_sure_site">
        <property name="text">
         <string>Sure</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_import_site">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_Frequency">
     <property name="title">
      <string>Frequencies</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_add_Fre">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_del_Fre">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QListView" name="listView"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_edit_Fre">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_export_Fre">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_sure_Fre">
        <property name="text">
         <string>Sure</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_import_Fre">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_import">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_sure">
     <property name="text">
      <string>Edit Finish</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_export">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
